errors/runtime/runtime-error.ts

```diff
@@ -0,0 +1,7 @@
+export class RuntimeError extends Error {
+  constructor(message: string) {

```

Comment:
I suggest to add details object:
```
constructor(message: string, public details?: Record<string, any>)
```
The details object contains details of error

Reply:
Does the details object need to join with the message?